pkg/util/test/util.go

```diff
@@ -2578,6 +2610,255 @@ func ValidateAlertManagerDisabled(pxImageList map[string]string, cluster *corev1
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry-metrics-collector deployment, waiting for deletion")
+		}
+
+		_, err = appops.Instance().GetDeployment("px-telemetry-registration", cluster.Namespace)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry-registration deployment, waiting for deletion")
+		}
+
+		_, err = appops.Instance().GetDaemonSet("px-telemetry-phonehome", cluster.Namespace)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry-phonehome daemonset, waiting for deletion")
+		}
+
+		// Verify telemetry roles are removed
+		_, err = rbacops.Instance().GetRole("px-telemetry", cluster.Name)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry role, waiting for deletion")
+		}
+
+		// Verify telemetry rolebindings
+		_, err = rbacops.Instance().GetRoleBinding("px-telemetry", cluster.Name)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry rolebinding, waiting for deletion")
+		}
+
+		// Verify telemetry configmaps
+		_, err = coreops.Instance().GetConfigMap("px-telemetry-collector", cluster.Namespace)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry-collector configmap, waiting for deletion")
+		}
+
+		_, err = coreops.Instance().GetConfigMap("px-telemetry-collector-proxy", cluster.Namespace)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry-collector-proxy configmap, waiting for deletion")
+		}
+
+		_, err = coreops.Instance().GetConfigMap("px-telemetry-phonehome", cluster.Namespace)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry-phonehome configmap, waiting for deletion")
+		}
+
+		_, err = coreops.Instance().GetConfigMap("px-telemetry-phonehome-proxy", cluster.Namespace)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry-phonehome-proxy configmap, waiting for deletion")
+		}
+
+		_, err = coreops.Instance().GetConfigMap("px-telemetry-register", cluster.Namespace)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry-register configmap, waiting for deletion")
+		}
+
+		_, err = coreops.Instance().GetConfigMap("px-telemetry-register-proxy", cluster.Namespace)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry-register-proxy configmap, wait for deletion")
+		}
+
+		_, err = coreops.Instance().GetConfigMap("px-telemetry-tls-certificate", cluster.Namespace)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry-tls-certificate configmap, waiting for deletion")
+		}
+
+		// Verify telemetry serviceaccounts
+		_, err = coreops.Instance().GetServiceAccount("px-telemetry", cluster.Namespace)
+		if !errors.IsNotFound(err) {
+			return "", true, fmt.Errorf("found px-telemetry serviceaccount, waiting for deletion")
+		}
+
```

Comment:
same here for clusterrole and clusterrolebinding

Reply:
Done